src/components/SortFeature.js

```diff
@@ -0,0 +1,34 @@
+import { IconButton, Menu, MenuButton } from "@chakra-ui/react";
+import {
+  StarIcon,
+  TimeIcon,
+  TriangleDownIcon,
+  TriangleUpIcon
+} from "@chakra-ui/icons";
+
+export const SortFeature = ({ sortOption, setSortOption }) => {
+  return (
+    <IconButton
+      position="absolute"
+      top="10px"
+      right="10px"
+      colorScheme="purple"
+      borderRadius="lg"
+      p="2"
```

Comment:
Que pensas de este estilado? Para hacerlo un poco más sutil pensé yo

```suggestion
      position="absolute"
      top="0px"
      right="0px"
      variant="link"
      colorScheme="purple"
      p="2"
```

Me gusta como queda con ese estilado, y hasta creo que le sacaría los iconos de las flechas